release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -156,6 +156,7 @@ With this release, there are several updates to the *Administrator* perspective
 
 * The {product-title} web console displays a `ConsoleNotification` if the cluster is upgrading. Once the upgrade is done, the notification is removed.
 * A *_restart rollout_* option for the `Deployment` resource and a *_retry rollouts_* option for the `DeploymentConfig` resource are available on the *Action* and *Kebab* menus.
+* You can view a list of supported clusters on the *All Clusters* dropdown list. The supported products include {product-title}, {product-title} Service on AWS (ROSA), Azure Red Hat OpenShift (ARO), ROKS, and {product-dedicated}. 
```

Comment:
A tiny nit that the first sentence has _supported clusters_, where the second has _supported products_. Should you use the same wording for clarity?

Reply:
Ah should be clusters. Thansk!